### ISSUE TITLE
StringIndexOutOfBounds when opening a project.

### ISSUE THREAD

**Issue by d-t-w** [cursiveclojure/cursive#958]
Intellij: 14.1.4
Cursive: 0.1.57

I regularly encounter this exception when opening a project, and oddly the IDE warning is in the top left-hand corner of the screen rather bubbling up from the bottom-right.

I can't easily reproduce it, appears to happen occasionally regardless of project being opened, and once it has happened will continue to trigger on every project opened until they have all been closed.

Not sure this is detailed enough to be helpful, will try and isolate further.

```
String index out of range: -1
java.lang.StringIndexOutOfBoundsException: String index out of range: -1
	at java.lang.String.substring(String.java:1911)
	at clojure.core$subs.invoke(core.clj:4781)
	at cursive.filters$fold_line_QMARK_.invoke(filters.clj:147)
	at cursive.filters$stacktrace_folder$fn__8063.invoke(filters.clj:157)
	at cursive.repl.StyledOutputBuffer.addFolding(StyledOutputBuffer.java:456)
	at cursive.repl.StyledOutputBuffer.updateFoldings(StyledOutputBuffer.java:388)
	at cursive.repl.StyledOutputBuffer.access$2700(StyledOutputBuffer.java:40)
	at cursive.repl.StyledOutputBuffer$Text.execute(StyledOutputBuffer.java:338)
	at cursive.repl.StyledOutputBuffer$1.run(StyledOutputBuffer.java:92)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:332)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:312)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:733)
	at java.awt.EventQueue.access$200(EventQueue.java:103)
	at java.awt.EventQueue$3.run(EventQueue.java:694)
	at java.awt.EventQueue$3.run(EventQueue.java:692)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:703)
	at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:734)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:569)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:382)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:242)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:161)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:150)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:146)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:138)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:91)
```

**Comment by cursiveclojure** [comment]
So this is based on something in your REPL output. It's a line that spuriously matches the regex used for the stacktrace folding. I believe this is actually the same error as #947.

**Comment by cursiveclojure** [comment]
I've fixed #947 for the next build. I believe this to be the same root cause, so I'm closing this - please let me know if you continue to see this behaviour.
